R/conjunct_test_linter.R

```diff
@@ -17,9 +17,11 @@ conjunct_expectation_linter <- function() {
 
     xml <- source_file$full_xml_parsed_content
 
+    # TODO(#1016): include assert_that() for consideration here too
```

Comment:
That would be just ` or text() = 'assert_that'` in the XPath

Reply:
right... plus tests. better not to clutter this PR with that